docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.5'
+
+services:
+  lmdeploy:
+    container_name: lmdeploy
+    image: openmmlab/lmdeploy-builder:cuda12.2
```

Comment:
`openmmlab/lmdeploy-builder` is ONLY for building whl packages. It is not an image to deploy a model

Reply:
i will change "openmmlab/lmdeploy-builder:cuda12.2" to "openmmlab/lmdeploy:latest"